pkgs/data/fonts/minecraftia/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchzip, stdenvNoCC }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "minecraftia";
+  version = "1.0";
```

Comment:
Where does this version come from?

Reply:
I checked the font's metadata using Birdfont. It is very possible that the author didn'tt care about versioning and went with 1.0 as the default. As far as I'm aware there are no other versions of this font.